features/extension/src/test/resources/wirings/5

```diff
@@ -0,0 +1,4 @@
+osgi.ee; (&(osgi.ee=JavaSE)(version=1.8))
```

Comment:
This file doesn't contain the license header. You should exclude from rat configuration.